test/jdk/com/sun/management/ThreadMXBean/ThreadAllocatedMemory.java

```diff
@@ -300,11 +302,16 @@ private static long checkResult(Thread curThread,
         return currSize;
     }
 
+    private static void reportUnexpected(Exception e, String reason) {
```

Comment:
nit: 'when' or 'where' seems a more appropriate name than 'reason'

Reply:
Changed to "when".